decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
Agree with you, but sei will drop by decodeNalu and never goes here. So the root cause should be no key frame after seek, right?